### PR TITLE
feat: autofill client Id and client secret for generate command

### PR DESCRIPTION
**Pull request by honcao**
**Reason for Change**:

feat: autofill client Id and client secret for generate command

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: cmd/generate.go [hunk 1]
```diff
@@ -132,7 +147,7 @@ func (gc *generateCmd) loadAPIModel() error {
 			Locale: gc.locale,
 		},
 	}
-	gc.containerService, gc.apiVersion, err = apiloader.LoadContainerServiceFromFile(gc.apimodelPath, true, false, nil)
```
Comment (jackfrancis) [inline cmd/generate.go]: The existing implementation runs a full ContainerService validation (above). The new one seems only to run a vlabs ContainerService validation. Is that intentional?

Comment (CecileRobertMichon) [reply cmd/generate.go]: this change is equivalent to one we made for deploy https://github.com/Azure/aks-engine/pull/1429/files

Comment (honcao) [reply cmd/generate.go]: that is correct, I followed the pattern in deploy command.

**Pull request merged**